src/main/java/Servlet/FormPageServlet.java

```diff
@@ -0,0 +1,16 @@
+package Servlet;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet("/")
+public class FormPageServlet extends HttpServlet {
```

Comment:
Why did you create separate servlet?
We could implement both `doGet` and `doPost` in one